CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 * Bump lodash to 4.17.14
 * Bump standard to 13.1.0
+* Add scroll percent to context
+* Add custom state to context
```

Comment:
because createContext takes `custom` as it's first param.. this will make it a breaking semver change yeah?

Reply:
Nah, the only external change is adding `custom` as the third (optional) param to `placementEngine`:

`function placementEngine (storage, promos, custom)`

Nothing changes if TC doesn't provide it.

Reply:
🤦‍♂